locale/sv/LC_MESSAGES/django.po

```diff
@@ -18,113 +18,89 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: home/models.py:54
 msgid "FAQs"
-msgstr ""
+msgstr "Vanliga frågor"
 
-#: home/templates/home/home_page.html:16 omahelsinki/templates/base.html:89
 msgid "Log in"
 msgstr "Logga in"
 
-#: omahelsinki/settings.py:142
 msgid "Finnish"
-msgstr ""
+msgstr "suomi"
```

Comment:
I always thought that `Finnish` in Swedish was `finska`. 😬 
Same goes for other langs.

Reply:
Haha good catch.